plugins/inventory/nb_inventory.py

```diff
@@ -1116,6 +1128,7 @@ def fetch_api_docs(self):
         )
 
         self.api_version = openapi["info"]["version"]
```

Comment:
I introduced a new variable `self.api_version_parsed` to optimize the code so that multiple conversions don't have to be done (on the lines 373-375 of nb_inventory.py). I didn't want to redefine the original variable because I wasn't sure if it is used elsewhere in the original string form.

Reply:
Remove L1131, make the suggested change above to L1130 and then just change where you changed to `self.api_version_parsed` to `self.api_version`. You parse the `openapi['info']['version']` once and then obviously any comparisons would need  `version.parse` objects to perform the comparisons against.